erpnext_com/www/docs/user/manual/en/accounts/opening-balance/fixed_assets.md

```diff
@@ -0,0 +1,37 @@
+<!-- add-breadcrumbs -->
+# Import Fixed Assets
+
+To import all the existing fixed assets first create the asset record and then create a journal entry to update general ledger.
+
+### 1. Create Asset Records
+
+Create Asset record for each asset that your company owns which is not fully depreciated.
```

Comment:
'which can be still used'  sounds simpler

Reply:
No, some assets are still usable but already fully depreciated. And as per my understanding there is no need to bring such assets into the new system.